pkgs/applications/audio/pd-plugins/gem/default.nix

```diff
@@ -0,0 +1,61 @@
+{ stdenv
+, fetchFromGitHub
+, fetchpatch
+, autoreconfHook
+, libtool
+, pkgconfig
+, puredata
+, libGL
+, libGLU
+, glew
+, freeglut
+, libv4l
+, libX11
+, file
+ }:
+
+stdenv.mkDerivation rec {
+  pname = "gem";
+  version = "unstable-2020-03-26";
+
+  src = fetchFromGitHub {
+    owner = "umlaeute";
+    repo = "Gem";
+    rev = "f38748d71bfca00e4d2b2f31d6c4e3759c03d599";
+    sha256 = "0bkky5fk0a836bapslrgzil272iq9y704y7hw254cfq5ffjd4qjy";
+  };
+
+  patches = [
+    # Update autoconf OpenGL/GLU/GLUT detection scripts
+    # https://github.com/umlaeute/Gem/pull/251
+    (fetchpatch {
+      url = "https://github.com/umlaeute/Gem/commit/343a486c2b5c3427696f77aeabdff440e6590fc7.diff";
+      sha256 = "0gkzxv80rgg8lgp9av5qp6xng3ldhnbjz9d6r7ym784fw8yx41yj";
+    })
+  ];
+
+  nativeBuildInputs = [
+    autoreconfHook
+    pkgconfig
+    puredata
```

Comment:
I don't know what "pure" is, but are you sure this should be here?

Reply:
'puredata' is the application that 'gem' is a plugin for. ./configure requires puredata to be available when building, but after building there doesn't appear to be anything in the gem package linking directly to puredata, which is why I figured it could be in `nativeBuildInputs`. Not sure though?

Reply:
Well the question is: whether it is a "build time _only_" dependency or a runtime dependency. The fact that it isn't linked "hardly" (as one might decide with `ldd`) doesn't mean that it's not a runtime dependency.

Since `puredata` will presumably be used on the machine that will _run_ `gem`, I'd conclude it should belong to `buildInputs`.